### PR TITLE
Update first example

### PR DESCRIPTION
**Pull request by kunicmarko20**
**Summary of Changes**

1. Add a line that makes first example work

For me first example didn't work until I added `ListenAndServe`, if this is not needed feel free to close the pr.

### CODE REVIEW

**Comment by fharding1** [comment]
Judging by the paragraph above:

```
Let's start registering a couple of URL paths and handlers:
```

It seems like this code block was only supposed to describe how to register the URL handlers and use gorilla/mux, not register them and start an HTTP server using the muxer, however, since it's including `func main()` it seems like it's supposed to be a full runnable example. In the latter case I think this change makes sense (and we should probably add import statements too), however maybe the `func main()` should just be excluded? This kind of reminds me of https://github.com/gorilla/mux/issues/506 . What do you think?

**Comment by elithrar** [comment]
Fully runnable examples are good, and there is one further down too.

**Comment by kunicmarko20** [comment]
Maybe make the first (or all examples) fully runnable since I didn't read the whole readme, my mistake. This change would probably remove confusion for new people.

**Comment by elithrar** [comment]
Making them all fully runnable obscures the examples - since the (best
practice) code to make them runnable is often longer than the example.

**Comment by kunicmarko20** [comment]
So. any thing you want me to update in this PR or should I just close it?

File: README.md [hunk 1]
```diff
@@ -55,6 +55,7 @@ func main() {
     r.HandleFunc("/products", ProductsHandler)
     r.HandleFunc("/articles", ArticlesHandler)
     http.Handle("/", r)
```
Comment (elithrar) [inline README.md]: Replace both these lines with:

```go
log.Fatal(http.ListenAndServe(":8080", r))`
```

I would also add a comment above the handlers, along the lines of:

```go
// Each HandleFunc takes a path to match and a http.HandlerFunc to call on that match
```

File: README.md [hunk 2]
```diff
@@ -55,6 +55,7 @@ func main() {
...
+    http.ListenAndServe(":8080", nil)
 }
```
Comment (elithrar) [inline README.md]: Replace both these lines with:

```go
log.Fatal(http.ListenAndServe(":8080", r))`
```

I would also add a comment above the handlers, along the lines of:

```go
// Each HandleFunc takes a path to match and a http.HandlerFunc to call on that match
```

**Comment by elithrar** [comment]
There is also https://github.com/gorilla/mux#full-example in the README

**Pull request closed without merging**